terraform/environment/modules/environment/rds.tf

```diff
@@ -132,6 +132,30 @@ resource "aws_db_parameter_group" "postgres-db-params" {
   }
 }
 
+resource "aws_db_parameter_group" "postgres13-db-params" {
+  name        = lower("postgres13-db-params-${var.environment_name}")
+  description = "default postgres13 rds parameter group"
+  family      = var.account.psql13_parameter_group_family
+  parameter {
+    name         = "log_min_duration_statement"
+    value        = "500"
+    apply_method = "immediate"
+  }
+
+  parameter {
+    name         = "log_statement"
+    value        = "all"
```

Comment:
Do we need to log all statements? I imagine that's going to produce massive logs.

Reply:
Although it will all SQL statements, it will only log it if the transaction takes longer than 500 ms (specified by log_min_duration_statement)